### ISSUE TITLE
Request some help for training GAMa-Net

### ISSUE THREAD

**Issue by Soyeon11** [svyas23/GAMa#3]
Hello, 
Can I get some hyper-parameter and batch size for trainging GAMa-Net & Hierachical model??
And if you have your checkpoint model for your paper, Can i get your checkpoint model?
I want to implement a model with the same recall rate as your paper.

**Comment by svyas23** [comment]
Hello Soyeon11

All hyperparameters are available in the code. Let me know if you are
unable to find something.
The model weights for final evaluation using GAMa-Net can be downloaded
here:
https://drive.google.com/file/d/1jmGpvWPISrlXRjEA6J_aCWwk2Jhnl-_J/view?usp=sharing

**Comment by Soyeon11** [comment]
Thank you for Reply! 

1. Did Result models in your paper use same hyper parameter in each main.py codes' default argument value??
I knew NT-Xent Loss performs good performance with large batchsize.
If you use small batch size such as 8or 16, Did you have any reason for small batch size?

2. If you have finetunned weights for fincal evaluation using GAMa-Net Screening_net, Can i get them?

Thank you for you help again.

**Comment by svyas23** [comment]
The weights shared are for final evaluation. We have used the largest batch size we could use with our system.
